stubs/table-view.stub

```diff
@@ -30,4 +30,14 @@ class DummyClass extends TableView
     {
         return [];
     }
+
+     /**
```

Comment:
I think it would be better to remove this from the stub file and add it to the documentation